### ISSUE TITLE
Duplicate x-axis in bar chart when axes are switched

### ISSUE THREAD

**Issue by grassick** [masayuki0812/c3#1292]
![image](https://cloud.githubusercontent.com/assets/1238335/8932779/cf61cfd4-3504-11e5-9e5f-55a81faf711c.png)

Worked in 0.4.10.

**Comment by daxfohl** [comment]
here's the offending commit:

https://github.com/masayuki0812/c3/commit/1168b6926d53eecc76f61fd4048ce1050aff306f#diff-a88a43467d0fb57fc0542b3e9341cdf2

subchart_axis_x_show is independent of subchart_show.  So it should be
```
-           .style("visibility", config.subchart_axis_x_show ? 'visible' : 'hidden');
+           .style("visibility", (config.subchart_show && config.subchart_axis_x_show) ? 'visible' : 'hidden');
```
unless it would be better to handle at a higher level.

**Comment by masayuki0812** [comment]
This should be fixed by #1294
